rules/cloud/azure/azure_ad_users_added_to_device_admin_roles.yml

```diff
@@ -0,0 +1,27 @@
+title: Users Added to Global or Device Admin Roles
+id: 11c767ae-500b-423b-bae3-b234450736ed
+description: Monitor and alert for users added to device admin roles.
+author: Michael Epping, '@mepples21'
+date: 2022/06/28
+references:
+  - https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/security-operations-devices#device-administrator-roles
+logsource:
+  product: azure
+  service: auditlogs
+detection:
+  selection:
+    Category: RoleManagement
+    OperationName|contains|all:
+      - 'Add'
+      - 'member to role'
+    TargetResources|contains:
+      - '7698a772-787b-4ac8-901f-60d6b08affd2'
+      - '62e90394-69f5-4237-9190-012177145e10'
+  condition: selection
```

Comment:
```
      - '7698a772-787b-4ac8-901f-60d6b08affd2'  # Cloud Device Administrator 
      - '62e90394-69f5-4237-9190-012177145e10' # Global Administrator
```
find in https://docs.microsoft.com/en-us/azure/active-directory/roles/permissions-reference